### ISSUE TITLE
U-Boot SPL

### ISSUE THREAD

**Issue by vpeter4** [UDOOboard/uboot-imx#1]
I am wondering if anyone thinks to support U-Boot SPL (Secondary Program Loader)? It would simplify using Udoo dual and quad with only one bootloader (same image for both devices).

Seems no interest that's why I will try to make it. Based on wandboard spl with the help of Fabio Estevam.

**Comment by vpeter4** [comment]
Not bad for a few days work :)
Quad done, DualLite is next.

``` javascript
U-Boot SPL 2015.07 (Aug 01 2015 - 01:29:52)


U-Boot 2015.07 (Aug 01 2015 - 01:29:52 +0200)

CPU:   Freescale i.MX6Q rev1.2 996 MHz (running at 792 MHz)
Reset cause: POR
Board: Udoo Quad
DRAM:  1 GiB
MMC:   FSL_SDHC: 0
*** Warning - bad CRC, using default environment

In:    serial
Out:   serial
Err:   serial
Net:   using phy at 6
FEC [PRIME]
Hit any key to stop autoboot:  0
=>
=> printenv board_type
board_type=quad
=>
```

**Comment by graugans** [comment]
@vpeter4 any updates on SPL for UDOO?

**Comment by vpeter4** [comment]
Yes, it is already available in mainline u-boot (2015.10-rc1 and up).
And used in Robert Nelson instructions: https://eewiki.net/display/linuxonarm/UDOO#UDOO-Bootloader:U-Boot
